docs/blog/2023-09-06-distributed-key-generation.md

```diff
@@ -0,0 +1,154 @@
+
+**🏴‍☠️ Reconstruction:** Based on a given threshold, the pirates can reconstruct the secret. What this eventually means is that a subset of pirates more or equal to a specific threshold t can combine their final aggregated shares to reconstruct the polynomial and thus the secret.
+
+![hooray.gif](./images/2023-09-06-dkg/hooray.gif)
+
+
+## Decentralization:
+
+The beauty of DKG is its decentralized nature. Since every pirate contributes equally to the randomness of the key, it's incredibly difficult for an outside adversary to determine the full secret key without access to the requisite number of shares. Moreover, the process guarantees that even if a pirate tries to cheat and distribute false shares they can be detected and potentially excluded from the protocol.
+
+Overall, DKG algorithms are an amazing research area and they can be used securely in a variety of cryptographic operations, like generating secret keys for signing or validating. Furthermore, by allowing DKG to be placed,  the power is spread among the participants, reducing the risk of a single point of failure and increasing the overall security of the network.
+
+# Drand’s DKG Approach:
+
+[Drand](https://drand.love/) incorporates [Pedersen's DKG scheme](https://www.cs.cornell.edu/courses/cs754/2001fa/129.PDF), an advanced framework originally devised by Torben Pryds Pedersen. This method ingeniously builds upon multiple instances of [Feldman's Verifiable Secret Sharing (VSS)](https://www.cs.umd.edu/~gasarch/TOPICS/secretsharing/feldmanVSS.pdf), enhancing them with added verification procedures to further secure the protocol.
+
+From a high-level perspective, the protocol can be divided into three distinct phases:
+
+1. **Share Distribution:**  Each participant generates a random secret and distributes it to the other participants using Feldman's VSS. Then, each party sends a unique share to every other member and publicly broadcasts a list of commitments.
+2. **Verification Phase**: Each participant verifies the received shares they have received from other participants. If a participant identifies an incorrect or invalid share, they issue a public complaint against the sender. The accused party must then either correct the share or risk being deemed an unreliable dealer.
+3. **Finalization Phase:** During the final phase, all verified shares from every participant are aggregated. However, any participant found to be unreliable or invalid during the verification stage is excluded from this aggregation
+
+********************************************Metrics of the protocol:********************************************
+
+| Network Model | Fault tolerance | Adaptive adversary | Discrete log key | CommunicationComplexity | Comp. Complexity | Round Complexity |
+| --- | --- | --- | --- | --- | --- | --- |
+| Sync. | 1/2 | No | ✅ | $O(n\mathcal{B}(\kappa t))$ | $O(nt^3)$ | $O(R)$ |
```

Comment:
Not sure I understand what is kappa and B and t here. Maybe O(n^2) is simpler no ? Also for computation complexity, why nt^3 ?  and round complexity, what is R ?